### ISSUE TITLE
Should we allow specific temperatures in Canto?

### ISSUE THREAD

**Issue by ValWood** [pombase/curation#3362]
Was part of the discussion for Maria's data.

If we do I this we have to report the specific temperature in the comments some times. So we might be able to identify those and convert. For example in
/pombe/curs/fcfd9a50a6a6cb53

**Comment by ValWood** [comment]
@manulera

**Comment by ValWood** [comment]
Hi @manu, what do you think about creating specific temperatures?

**Comment by manulera** [comment]
Yes, I think we could do this in FYECO. The only thing is that this is only useful instead of "standard temperature" if the phenotype is absent in the same study at another standard temperature, say the same study shows different phenotypes at 25 and 30.

**Comment by ValWood** [comment]
I think that is OK. At least we have the option to differentiate when necessary.

**Comment by manulera** [comment]
cc @kimrutherford @ValWood 

Related to https://github.com/pombase/curation/issues/3465

For now in the HTP files what I did was to use the FYECO term and put the temperature in parenthesis. I guess this information is currently discarded when loading into canto. Maybe it's a good moment to discuss about this next week. Kind of tedious decision to make I guess, but at least we have recently discussed about it for the HTP files.
